app/assets/stylesheets/application.css

```diff
@@ -10,13 +10,13 @@
  * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
  * file per style scope.
  *
- *= require_tree .
  *= require dataTables/jquery.dataTables
```

Comment:
Likewise this one.

We should probably add in the bs3 style for now, right?

Reply:
Yeah, I found it. I can switch it over when needed.
It's funny--I'm only noticing these mistakes during testing because I still have the gems installed 🙄

Reply:
Whelp it looks terrible.

Reply:
Used default styling, is much nicer.

Reply:
>  Used default styling, is much nicer.

I suspect you missed something (like perhaps the `table` class?) but also, whatever. We'll worry about that for BS4